gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java

```diff
@@ -3,43 +3,26 @@
 import static com.google.common.base.Preconditions.checkArgument;
 
 import com.google.auto.value.AutoValue;
+import com.google.common.base.Strings;
 
 import java.util.Map;
 
+import javax.annotation.Nullable;
+
 /**
  * Configuration for {@link CloudStorageFileSystem} instances.
  */
 @AutoValue
 public abstract class CloudStorageConfiguration {
 
-  /**
-   * Returns path of current working directory. This defaults to the root directory.
-   */
-  public abstract String workingDirectory();
-
-  /**
-   * Returns {@code true} if we <i>shouldn't</i> throw an exception when encountering object names
-   * containing superfluous slashes, e.g. {@code a//b}.
-   */
-  public abstract boolean permitEmptyPathComponents();
-
-  /**
-   * Returns {@code true} if '/' prefix on absolute object names should be removed before I/O.
-   *
-   * <p>If you disable this feature, please take into consideration that all paths created from a
-   * URI will have the leading slash.
-   */
-  public abstract boolean stripPrefixSlash();
-
-  /**
-   * Returns {@code true} if paths with a trailing slash should be treated as fake directories.
-   */
-  public abstract boolean usePseudoDirectories();
+  private static final CloudStorageConfiguration DEFAULT = builder().build();
 
   /**
-   * Returns block size (in bytes) used when talking to the GCS HTTP server.
+   * Returns default GCS NIO configuration.
    */
-  public abstract int blockSize();
+  public static CloudStorageConfiguration getDefault() {
```

Comment:
Nit: maybe `defaultInstance`? If we care about being consistent with our `<Service>Options`.defaultInstance()`